app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -75,6 +116,12 @@ export class ArticleCoverImage extends Component {
     const { mainImage } = this.props;
     const { uploadError, uploadErrorMessage, uploadingImage } = this.state;
     const uploadLabel = mainImage ? 'Change' : 'Add a cover image';
+    const extraProps = this.useNativeUpload()
```

Comment:
Documentation (blocking): Add a small explanation of what's happening here. I know we discussed it on the Zoom call yesterday, but it would be good to be explicit as to what's going on, i.e. the tabindex="-1" etc.

Reply:
Makes sense 👍  

I just added some context inline to explain the reasoning behind the `extraProps` in both `ArticleCoverImage` and `ImageUploader`. Let me know if it reads okay or theres anything missing :)